mods/coop/hook/lua/ScenarioFramework.lua

```diff
@@ -118,7 +134,16 @@ end
 -- @param failureDialogue A VO to play explaining how much of a failure you are (if any).
 -- @param currentObjectives The AssignedObjectives from the map.
 function PlayerDeath(deadCommander, failureDialogue, currentObjectives)
-    if ScenarioInfo.OpEnded then
+    if failureDialogue then
+        FailDialogue = failureDialogue
```

Comment:
Why do you create a new global variable here? This new IF conditions isnt changing anything

Reply:
This saves a message for use when all ACUs are destroyed.